src/components/todo/TodoComponent.tsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import { useQueryClient } from '@tanstack/react-query';
-import { useUpdateTodoTime } from '@/app/apiTest/hooks/todoQueries';
+import { useUpdateTodoTime } from '@/api/hooks/todoHooks';
 import * as S from './Todo.styled';
 import { Text } from '../common';
 
 interface TodoComponentProps {
   todoId: number;
   title: string;
-  setTodoId: (s: string) => void;
+  setTodoId: (s: number) => void;
 
   setIsModalOpenTrue: () => void;
   setIsModalOpenedByFABFalse: () => void;
```

Comment:
```suggestion
  setIsModalOpenTrue: VoidFunction;
  setIsModalOpenedByFABFalse: VoidFunction;
```
p4: 이렇게 쓰는 방식도 있답니다!

Reply:
그런 방법이 있었군요! 조언 감사합니다:)